index.js

```diff
@@ -1,4 +1,40 @@
-export default function uniqueRandom(minimum, maximum) {
+function _exhaustive(minimum, maximum) {
+	function integersInRange(minimum, maximum) {
```

Comment:
This function could be moved outside of the function. I'm not sure whether it would be cleaner as a generator.